gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc

```diff
@@ -0,0 +1,380 @@
+= Renewing Flex Gateway Registration
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+== Using CLI Commands to Renew and Verify Registration
+
+Flex Gateway uses a managed PKI certificate to communicate with Anypoint Platform. This certificate is generated when you register Flex Gateway in either Connected Mode or Local Mode. Updating the certificate ensures the continued operation of your applications. You update a certificate by renewing registration, via the `flexctl registration renew` CLI.
+
+Attempting to run Flex Gateway with an expired certificate in Connected Mode results in the following:
+
+* New replicas fail to download API configurations from Anypoint Platform, thereby becoming unusable. Existing clusters fail to reload.
+* Logs and metrics fail to upload to Anypoint Platform. Troubleshooting using Anypoint Platform is not possible.
+* Metering information fails to upload to Anypoint Platform. MuleSoft is unable to collect or report usage metrics.
+
+Attempting to run Flex Gateway with an expired certificate in Local Mode results in the following:
+
+* Metering information fails to upload to Anypoint Platform. MuleSoft is unable to collect or report usage metrics.
+
+If a certificate expires, you can still invoke `flexctl registration renew`, which updates the expired certificate.
+
+To avoid disruptions:
+
+. <<before-you-begin,Download the latest version of Flex Gateway.>>
+. <<renewal-cli-overview,Renew your Flex Gateway registration.>>
+. <<inspection-cli-overview,Verify the success of the renewal process.>>
+
+[[before-you-begin]]
+=== 1. Download Latest Version of Flex Gateway
+
+To invoke the registration renewal `flexctl` command, download the latest version of Flex Gateway. You don't need to register and run this version of the gateway.
+
+* xref:flex-install.adoc[]
+
+[[renewal-cli-overview]]
+=== 2. Renew Your Registration
+
+Flex Gateway includes the following CLI command to renew registration:
+
+----
+flexctl registration renew
+----
+
+The command renews an existing Flex Gateway registration with Runtime Manager. For authentication, you run the command with either user credentials or with connected app credentials. Authentication with user credentials requires the `--username` and `--password` flags. Authentication with connected app credentials requires the `--client-id` and `--client-secret` flags.
```

Comment:
This is a paragraph under step 2.

Reply:
Fixed! I moved this contextual information to the page's introduction.